hpx/runtime/serialization/container.hpp

```diff
@@ -31,7 +31,7 @@ namespace hpx { namespace serialization
             naming::gid_type const & split_gid) = 0;
         virtual void set_filter(binary_filter* filter) = 0;
         virtual void save_binary(void const* address, std::size_t count) = 0;
```

Comment:
Couldn't we do the same here as you did for the chunk? Let `save_binary` return the number of bytes written?